libraries/AP_RCProtocol/AP_RCProtocol_IBUS.cpp

```diff
@@ -20,10 +20,12 @@
 
 #include "AP_RCProtocol_IBUS.h"
 
+static int ia6_hack = 0;
```

Comment:
Better to have an `AP_RCProtocol_IBUS_IA6` subclass so that other users can use this modified decoder by using the custom build server.